bin/tessel-2.js

```diff
@@ -277,6 +277,15 @@ parser.command('update')
   })
   .help('Update the Tessel firmware and openWRT image');
 
+parser.command('version')
+  .option('timeout', timeoutOption)
+  .option('name', nameOption)
```

Comment:
sure.  I made a little factory function, that we could use.

```
function tesselCommandFactory(commandName) {
  return parser.command(commandName)
    .option('timeout', timeoutOption)
    .option('name', nameOption)
    .option('lan', {
      flag: true,
      help: 'Use LAN connection'
    })
    .option('usb', {
      flag: true,
      help: 'Use USB connection'
    });
}

tesselCommandFactory('version')
  .callback(function(opts) {
    controller.tesselFirmwareVerion(opts)
      .then(closeSuccessfulCommand, closeFailedCommand);
  })
  .help('Display Tessel\'s current firmware version');
```

can make another PR for that if you like?